rcl_action/src/rcl_action/action_client.c

```diff
@@ -230,6 +245,10 @@ rcl_action_client_init(
   SUBSCRIPTION_INIT(feedback);
   SUBSCRIPTION_INIT(status);
 
+  // The resolved action name is no longer needed
+  allocator.deallocate(resolved_action_name, allocator.state);
+  resolved_action_name = NULL;
+
```

Comment:
same comment, this can be removed if we do not need `resolved_action_name` in stack memory.